scalecodec/type_registry/polymesh.json

```diff
@@ -1,5 +1,14 @@
 {
   "types": {
+    "AccountInfo": "AccountInfoWithRefCount",
+    "Address": "IndicesLookupSource",
+    "LookupSource": "IndicesLookupSource",
```

Comment:
These changes were added by Mudit to solve some problems with PolkadotJS API latest version. 
I couldn't find `AccountInfoWithRefCount` and `IndicesLookupSource` defined in `default.json`.
My guess is that they would be required if we update scalecodec to upstream, making it compatible with latest Substrate version which we are not using yet.
I recommend you to skip them by now.